app/components/Resources/ResourcesTable.js

```diff
@@ -56,7 +58,7 @@ class ResourcesTable extends Component {
 		let page = this.state.page + 1;
 		this.setState({
 			page: page,
-			currentPage: this.state.resources.slice(page * 9 + 1, page * 9 + 10)
+			currentPage: this.state.resources.slice(page * resultsPerPage, page * resultsPerPage + (resultsPerPage + 1))
```

Comment:
`slice`'s `end` parameter is exclusive, right? For page zero, the call will be `this.state.resources.slice(0, 10)`, which returns resources indexed 0-9. That's ten resources. Do we actually want nine?